src/components/TodoList/TodoList.tsx

```diff
@@ -1,210 +1,117 @@
-              </button>
-            </td>
-          </tr>
-
-          <tr data-cy="todo">
-            <td className="is-vcentered">9</td>
-            <td className="is-vcentered"> </td>
-
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-danger">molestiae perspiciatis ipsa</p>
-            </td>
-
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye" />
-                </span>
-              </button>
-            </td>
-          </tr>
-
-          <tr data-cy="todo">
-            <td className="is-vcentered">10</td>
-
-            <td className="is-vcentered">
-              <span className="icon" data-cy="iconCompleted">
-                <i className="fas fa-check" />
-              </span>
-            </td>
-
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-success">illo est ratione doloremque quia maiores aut</p>
-            </td>
-
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye" />
-                </span>
-              </button>
-            </td>
-          </tr>
-        </tbody>
-      </table>
+  return (
+    <>
+      {!filteredTodos.length && query
+        ? (
+          <p className="notification is-warning">
+            There are no todos matching current filter criteria
+          </p>
+        ) : (
+          <table className="table is-narrow is-fullwidth">
+            <thead>
+              <tr>
+                <th>#</th>
+
+                <th>
+                  <span className="icon">
+                    <i className="fas fa-check" />
+                  </span>
+                </th>
+
+                <th>Title</th>
+                <th> </th>
+              </tr>
+            </thead>
+
+            <tbody>
+              {filteredTodos.map(todo => (
+                <tr data-cy="todo" key={todo.id}>
+                  <td className="is-vcentered">{todo.id}</td>
+                  {!todo.completed
+                    ? <td className="is-vcentered" />
+                    : (
+                      <td className="is-vcentered">
+                        <span className="icon" data-cy="iconCompleted">
+                          <i className="fas fa-check" />
+                        </span>
+                      </td>
+                    )}
+                  <td className="is-vcentered is-expanded">
+                    <p
+                      className={
+                        classNames({
+                          'has-text-danger': !todo.completed,
+                          'has-text-success': todo.completed,
+                        })
+                      }
```

Comment:
You can make this short a bit
just `import cn from 'classnames'`
```suggestion
                      className={cn({
                          'has-text-danger': !todo.completed,
                          'has-text-success': todo.completed,
                        })
                      }
```